docs/release_notes/release_notes.md

```diff
@@ -12,6 +12,12 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
 {% endif %}
 
+### Project Versions
+
+* Foreman 1.7
+* Pulp 2.5
+* Candlepin 0.9
+
 ## Upgrade Instructions
 
 ### Capsule
```

Comment:
This might be confusing, as this is already on the actual upgrade pages.  It might be worth the release note, but I'd say something about viewing the full upgrade instructions and a link to them